src/SpineAnimationRenderer.ts

```diff
@@ -269,6 +298,7 @@ export class SpineAnimationRenderer extends Renderer {
       _primitive.destroy();
     }
     this._clearMaterialCache();
+    this._resource && this._resource.destroy();
     this._primitive = null;
```

Comment:
_onDestroy should be more concise, like:
```typescript
override _onDestroy(): void {
  this._clearMaterialCache();
  this._subPrimitives.length = 0;
  this._primitive && this._primitive.destroy();
  this._resource && this._resource.destroy();
  this._primitive = null;
  this._resource = null;
  this._skeleton = null;
  this._state = null;
  super._onDestroy();
}
```

Reply:
done